src/components/Quiz/useQuizStore.ts

```
@@ -1,14 +1,32 @@
 import { create } from 'zustand';
 import { devtools } from 'zustand/middleware';
+import { useAuth } from '../Auth/Auth';
```

ESLint

[error] 'useAuth' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)